mlir/test/Dialect/Linalg/transform-op-specialize_matmul.mlir

```diff
@@ -0,0 +1,148 @@
+// RUN: mlir-opt --transform-interpreter --split-input-file --verify-diagnostics %s | FileCheck %s
+
+#map = affine_map<(d0, d1, d2) -> (d0, d2)>
+#map1 = affine_map<(d0, d1, d2) -> (d2, d1)>
+#map2 = affine_map<(d0, d1, d2) -> (d0, d1)>
+func.func @specialize_matmul(%arg0: tensor<?x?xf32>, %arg1: tensor<?x?xf32>, %arg2: tensor<?x?xf32>) -> tensor<?x?xf32> {
```

Comment:
Can you add element wise for transforms, too?

Reply:
It was added separately when i did elemwise specialization : look for  transform-op-specialize_elemwise_*.mlir